index.html

```diff
@@ -30,147 +30,148 @@
-                </ul>
-                <button class="card-button" type="submit">See Project</button>
-      </div>
-          
-    </section> 
-    <section class="card undefined">  
-        <img src="images/portfolio3.png" class="img-i" alt="Snapshoot-Portfolio">
-           
-            <div class="card-block" id="arti-2">
-                <h2>Tonic</h2>
-                <ul class="card-frame">
-                    <li class="no-count">Canopy</li>
-                    <li>Full Stack Dev</li>
-                    <li>2015</li>
-                </ul>
-                <p>Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.</p>
-                <ul class="card-tags">
-                    <li>html</li>
-                    <li>Javascript</li>
-                    <li>css</li>
-                </ul>
-                <button class="card-button" type="submit">See Project</button>
-      </div>
-          
-    </section> 
-    <section class="card undefined">  
-        <img src="images/portfolio4.png" class="img-i" alt="Snapshoot-Portfolio">
-           
-            <div class="card-block" id="arti-2">
-                <h2>Multi-post Stories</h2>
-                <ul class="card-frame">
-                    <li class="no-count">Canopy</li>
-                    <li>Full Stack Dev</li>
-                    <li>2015</li>
-                </ul>
-                <p>Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.</p>
-                <ul class="card-tags">
-                    <li>html</li>
-                    <li>Javascript</li>
-                    <li>css</li>
-                </ul>
-                <button class="card-button" type="submit">See Project</button>
-      </div>
-          
-    </section> 
+        <section id="works-container">
+                <div class="card1">
+                    <img src="images/portfolio1.png" class="img-i" alt="Snapshoot-Portfolio">
+                     
+                      <div class="card-block">
+                        <h2> Tonic</h2>
+                          <ul class="card-frame">
+                              <li class="no-count">CANOPY</li>
+                              <li>Back End Dev</li>
+                              <li>2015</li>
+                          </ul>
+                          <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                          <ul class="card-tags">
+                              <li>html</li>
+                              <li>css</li>
+                              <li>javaScript</li>
+                          </ul>
+                          <button class="card-button" type="submit">See Project</button>
+                     </div>
+                </div>
+            <div class="card undefined">  
```

Comment:
- [Optional] Please use an indentation of two spaces in all your documents. This is a convention and also [best practice](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).